src/Microsoft.OData.Edm/Schema/CoreModel/EdmCoreModelPrimitiveType.cs

```diff
@@ -0,0 +1,55 @@
+//---------------------------------------------------------------------
+// <copyright file="EdmCoreModelPrimitiveType.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+namespace Microsoft.OData.Edm
+{
+    /// <summary>
+    /// The built-in Edm.PrimitiveType and other concrete primitive types in the core model.
+    /// </summary>
+    internal sealed class EdmCoreModelPrimitiveType : EdmType, IEdmPrimitiveType, IEdmCoreModelElement
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="EdmCoreModelPrimitiveType"/> class.
+        /// </summary>
+        /// <param name="name">The name.</param>
+        /// <param name="pathKind">The path kind.</param>
+        public EdmCoreModelPrimitiveType(string name, EdmPrimitiveTypeKind primitiveKind)
+        {
+            Name = name ?? string.Empty;

```

Comment:
>?? string.Empty [](start = 24, length = 15)

Can name be empty?

Reply:
That's the existing codes copied from "EdmCoreModel.cs". But, I don't think the name can be empty. Let's me think it more to resolve it.